examples/c-api/buildsystem/main.c

```diff
@@ -179,8 +179,7 @@ static void command_process_had_output(void* context,
 static void command_process_finished(void* context,
                                      llb_buildsystem_command_t* command,
                                      llb_buildsystem_process_t* process,
-                                     llb_buildsystem_command_result_t result,
-                                     int exit_status) {
+                                     llb_buildsystem_command_extended_result_t result) {
```

Comment:
I think we want to pass this as a pointer. It might be more efficient, but more importantly it means we can extend the struct without changing the ABI.